docs/release-notes/0.2.0.md

```diff
@@ -43,25 +43,24 @@ Here are some configuration examples for each supported engine type:
 
 **MySQL**
 
-```yaml
+```sh
 [mysqld]
 key_buffer_size=16M
 max_allowed_packet=128M
 max_connections=250
 ```
 
-
 **Mongo**
 
-```yaml
+```sh
 operationProfiling:
 mode: slowOp
 slowOpThresholdMs: 200
```

Comment:
I know this isn't related to the install/uninstall that motivates this PR but the indentation isn't correct here:
it should be:
```yaml
operationProfiling:
  mode: slowOp
  slowOpThresholdMs: 200
```